nwbwidgets/base.py

```diff
@@ -370,3 +370,17 @@ def row_to_hover_text(row):
         elif isinstance(val, float):
             text_rows.append("{}: {:.3f}".format(key, val))
     return "<br>".join(text_rows)
+
+
+def show_multi_container_interface(
```

Comment:
okay, it does show up all the datainterfaces within the multicontainer, but does not show any labels.
So currently, nwbfile.processing['ecephys']['LFP']['A001'/'B001'..]. The accordian for 'processing' directly shows the 'A001','B001'. I think there should be another accordian for LFP
![image](https://user-images.githubusercontent.com/11567561/117231101-25c6fa00-ae3c-11eb-89fe-73c9008f0962.png)

Reply:
or the other possibility is to leave this as it is, since by making the 109 line deletion in the below [suggestion](https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets/pull/156#discussion_r627029939) populates the accordian title with the description, which is self-explanatory

Reply:
Oh, that's intentional. If an accordion as only a single option, I just go straight through to that option. So since LFP is the only object within ecephys, I just assume that's the one you'd want to select and forward you on to the contents of LFP